.github/CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ media: my-image-for-the-issue (make sure to place it on `/assets/img` and use th
 Here you can add any Markdown-valid content describing the accessibility issue.
 ```
 
-> :warning: Your tag must be one of the following: `"Non-perceivable"`, `"Non-operable"`, `"Non-understandable"`. To know which one to use, refer to the [POUR principles](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head).
+> :warning: Your tag must be one of the following: `"Non-perceivable"`, `"Non-operable"`, `"Non-understandable"`. To know which one to use, refer to the [FOUR principles](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head).
```

Comment:
This is supposed to be `POUR` not `FOUR`. :)

It means **P**erceivable, **O**perable, **U**nderstandable and **R**obust.